waspc/data/Generator/templates/Dockerfile

```diff
@@ -45,7 +44,8 @@ RUN apk add --no-cache python3
 ENV NODE_ENV production
 WORKDIR /app
 COPY --from=server-builder /app/server/node_modules ./server/node_modules
-COPY server/ ./server/
+COPY --from=server-builder /app/server/dist ./server/dist
+COPY --from=server-builder /app/server/package*.json ./server/
```

Comment:
In addition to the `dist` dir, we also need the `package.json` files to run the `npm run` command.